dpe/src/x509.rs

```diff
@@ -1323,5 +1404,14 @@ mod tests {
             Ok(None) => panic!("key usage extension not found"),
             Err(_) => panic!("multiple key usage extensions found"),
         }
+
+        match cert.extended_key_usage() {
+            Ok(Some(ext_key_usage)) => {
+                assert!(ext_key_usage.critical);
+                assert_eq!(ext_key_usage.value.other, [oid!(2.23.133 .8 .7)]);
```

Comment:
nit: Can you add a comment of which OID this is?

Reply:
Done.